lib/reverse_string.rb

```diff
@@ -1,5 +1,30 @@
 def reverse_characters(str)
+
+    i = 0    
+    n = str.size - 1
+    ansstr = " "*n
```

Comment:
Nice solution! If on line 5, you had declared `ansstr` as an empty string (`ansstr = ""`), how would you change the code within your `while` loop?